CRITICAL - 2025-03-17 18:40:45 --> CodeIgniter\Security\Exceptions\SecurityException: The action you requested is not allowed.
[Method: POST, Route: login]
in SYSTEMPATH/Security/Security.php on line 262.
 1 SYSTEMPATH/Security/Security.php(262): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
 2 SYSTEMPATH/Filters/CSRF.php(52): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
 3 SYSTEMPATH/Filters/Filters.php(205): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 4 SYSTEMPATH/Filters/Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 5 SYSTEMPATH/CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('login', 'before')
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 18:49:00 --> CodeIgniter\Filters\Exceptions\FilterException: "auth" filter must have a matching alias defined.
[Method: GET, Route: admin]
in SYSTEMPATH/Filters/Filters.php on line 495.
 1 SYSTEMPATH/Filters/Filters.php(495): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('auth')
 2 SYSTEMPATH/Filters/Filters.php(547): CodeIgniter\Filters\Filters->enableFilter('auth', 'before')
 3 SYSTEMPATH/CodeIgniter.php(469): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 18:52:09 --> CodeIgniter\Filters\Exceptions\FilterException: "auth" filter must have a matching alias defined.
[Method: GET, Route: admin]
in SYSTEMPATH/Filters/Filters.php on line 495.
 1 SYSTEMPATH/Filters/Filters.php(495): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('auth')
 2 SYSTEMPATH/Filters/Filters.php(547): CodeIgniter\Filters\Filters->enableFilter('auth', 'before')
 3 SYSTEMPATH/CodeIgniter.php(469): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 20:56:52 --> CodeIgniter\Security\Exceptions\SecurityException: The action you requested is not allowed.
[Method: POST, Route: login]
in SYSTEMPATH/Security/Security.php on line 262.
 1 SYSTEMPATH/Security/Security.php(262): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
 2 SYSTEMPATH/Filters/CSRF.php(52): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
 3 SYSTEMPATH/Filters/Filters.php(205): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 4 SYSTEMPATH/Filters/Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 5 SYSTEMPATH/CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('login', 'before')
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 20:58:42 --> CodeIgniter\Security\Exceptions\SecurityException: The action you requested is not allowed.
[Method: POST, Route: login]
in SYSTEMPATH/Security/Security.php on line 262.
 1 SYSTEMPATH/Security/Security.php(262): CodeIgniter\Security\Exceptions\SecurityException::forDisallowedAction()
 2 SYSTEMPATH/Filters/CSRF.php(52): CodeIgniter\Security\Security->verify(Object(CodeIgniter\HTTP\IncomingRequest))
 3 SYSTEMPATH/Filters/Filters.php(205): CodeIgniter\Filters\CSRF->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 4 SYSTEMPATH/Filters/Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 5 SYSTEMPATH/CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('login', 'before')
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 21:00:04 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'solvam' in /var/www/html/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('luigi-mariadb', 'root', 'solvam', 'solvam', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#7 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
#8 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#9 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database '****' in /var/www/html/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#6 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
#7 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#8 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-03-17 21:00:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database '****'
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Usuarios`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 21:04:21 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'solvam' in /var/www/html/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('luigi-mariadb', 'root', 'solvam', 'solvam', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#7 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
#8 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#9 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database '****' in /var/www/html/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#6 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
#7 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#8 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-03-17 21:04:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database '****'
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Usuarios`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '33333')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 21:04:47 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'solvam' in /var/www/html/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('luigi-mariadb', 'root', 'solvam', 'solvam', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#7 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '3333')
#8 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#9 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database '****' in /var/www/html/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#6 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '3333')
#7 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#8 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-03-17 21:04:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database '****'
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Usuarios`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('admin', '3333')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 21:05:02 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'solvam' in /var/www/html/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(187): mysqli->real_connect('luigi-mariadb', 'root', 'solvam', 'solvam', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#7 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('luigi', '1234')
#8 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#9 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database '****' in /var/www/html/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/system/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/system/BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 /var/www/html/app/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
#6 /var/www/html/app/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('luigi', '1234')
#7 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#8 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-03-17 21:05:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database '****'
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Usuarios`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH/Models/UsuariosModel.php(25): CodeIgniter\BaseModel->first()
 6 APPPATH/Controllers/Usuarios.php(296): App\Models\UsuariosModel->checkUser('luigi', '1234')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 21:06:55 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: login]
in SYSTEMPATH/BaseModel.php on line 826.
 1 SYSTEMPATH/BaseModel.php(826): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH/Controllers/Usuarios.php(335): CodeIgniter\Model->insert([...])
 4 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin('4', true)
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 21:07:15 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: login]
in SYSTEMPATH/BaseModel.php on line 826.
 1 SYSTEMPATH/BaseModel.php(826): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH/Controllers/Usuarios.php(335): CodeIgniter\Model->insert([...])
 4 APPPATH/Controllers/Usuarios.php(316): App\Controllers\Usuarios->logLogin(null, false)
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 21:47:36 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: login]
in SYSTEMPATH/BaseModel.php on line 826.
 1 SYSTEMPATH/BaseModel.php(826): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH/Controllers/Usuarios.php(328): CodeIgniter\Model->insert([...])
 4 APPPATH/Controllers/Usuarios.php(305): App\Controllers\Usuarios->logLogin('4', true)
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 21:49:55 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: login]
in SYSTEMPATH/BaseModel.php on line 826.
 1 SYSTEMPATH/BaseModel.php(826): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH/Controllers/Usuarios.php(326): CodeIgniter\Model->insert([...])
 4 APPPATH/Controllers/Usuarios.php(303): App\Controllers\Usuarios->logLogin('4', true)
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:11:21 --> mysqli_sql_exception: Column 'UsuarioID' cannot be null in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(NULL, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:11:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:11:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:11:21\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:11:21\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:11:22 --> [Caused by] mysqli_sql_exception: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:11:21\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:11:21\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:11:21\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:12:30 --> mysqli_sql_exception: Column 'UsuarioID' cannot be null in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(NULL, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:12:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:12:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:12:30\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:12:30\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:12:30 --> [Caused by] mysqli_sql_exception: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:12:30\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:12:30\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:12:30\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:15:25 --> mysqli_sql_exception: Column 'UsuarioID' cannot be null in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(NULL, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:15:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:15:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:15:25\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:15:25\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:15:25 --> [Caused by] mysqli_sql_exception: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:15:25\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:15:25\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:15:25\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:18:22 --> mysqli_sql_exception: Column 'UsuarioID' cannot be null in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(NULL, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:18:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:18:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:18:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:18:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:18:22 --> [Caused by] mysqli_sql_exception: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:18:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:18:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:18:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(333): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:21:19 --> mysqli_sql_exception: Column 'UsuarioID' cannot be null in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(336): App\Models\LoginLogModel->logLoginAttempt(NULL, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:21:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(336): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:21:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:21:19\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:21:19\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(336): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:21:19 --> [Caused by] mysqli_sql_exception: Column 'UsuarioID' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:21:19\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:21:19\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (NULL, \'2025-03-17 23:21:19\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(336): App\Models\LoginLogModel->logLoginAttempt(null, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:25:45 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`)) in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(-1, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:25:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(-1, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:25:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (-1, \'2025-03-17 23:25:45\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (-1, \'2025-03-17 23:25:45\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(-1, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:25:45 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (-1, \'2025-03-17 23:25:45\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (-1, \'2025-03-17 23:25:45\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (-1, \'2025-03-17 23:25:45\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(-1, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:32:22 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`)) in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:32:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:32:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:32:22 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:22\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:32:39 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`)) in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:32:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:32:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:39\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:39\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:32:39 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:39\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:39\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:32:39\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-17 23:33:32 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`)) in /var/www/html/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `Lo...', 0)
#1 /var/www/html/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Lo...')
#2 /var/www/html/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Lo...')
#3 /var/www/html/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Lo...', Array, false)
#4 /var/www/html/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/system/BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/system/Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/app/Models/LoginLogModel.php(47): CodeIgniter\Model->insert(Array)
#8 /var/www/html/app/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Wi...', 1, 'Credenciales in...')
#9 /var/www/html/app/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(NULL, false)
#10 /var/www/html/system/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
#11 /var/www/html/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 /var/www/html/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /var/www/html/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#14 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#15 /var/www/html/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-03-17 23:33:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 7 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:33:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:33:32\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:33:32\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
 9 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-03-17 23:33:32 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`solar_screen`.`LoginLog`, CONSTRAINT `LoginLog_ibfk_1` FOREIGN KEY (`UsuarioID`) REFERENCES `Usuarios` (`ID`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:33:32\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:33:32\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (0, \'2025-03-17 23:33:32\', 0, \'172.18.0.1\', \'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36\', 1, \'Credenciales incorrectas\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `LoginLog` (`UsuarioID`, `FechaHora`, `success`, `ip_address`, `user_agent`, `attempts`, `reason`) VALUES (:UsuarioID:, :FechaHora:, :success:, :ip_address:, :user_agent:, :attempts:, :reason:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginLogModel.php(47): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/Usuarios.php(332): App\Models\LoginLogModel->logLoginAttempt(0, false, '172.18.0.1', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36', 1, 'Credenciales incorrectas')
10 APPPATH/Controllers/Usuarios.php(310): App\Controllers\Usuarios->logLogin(null, false)
11 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Usuarios->checkUser()
12 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
13 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
14 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
15 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
16 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
